src/config.js

```diff
@@ -89,7 +89,12 @@ const context = {
 	GITHUB_REPOSITORY: getVar({
 		key: 'GITHUB_REPOSITORY',
 		required: true
-	})
+	}),
+	SKIP_PR: getVar({
```

Comment:
As the option only accepts true or false, specify the type like this:

```
type: 'boolean'
```

Reply:
Updated